src/components/AuthContainer.ts

```diff
@@ -28,7 +28,7 @@ const AuthContainer = ({ children }) => {
     [displayTosPage && name !== 'privacy', () => h(TermsOfService)],
     [registrationStatus === userStatus.disabled, () => h(Disabled)],
     [_.isEmpty(profile) && !isPublic, authspinner],
-    () => children
+    [true, () => children]
```

Comment:
The previous style was actually important for the type signature of `cond`. Removing the default value changes the return type of `cond` from `Type` to `Type | undefined`. It's not smart enough to recognize `[true, () => value]` as a default case.